integration/ig/k8s/trace_dns_test.go

```diff
@@ -24,13 +24,15 @@ import (
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
-func TestTraceDns(t *testing.T) {
-	t.Parallel()
-	ns := GenerateTestNamespaceName("test-trace-dns")
+func newTraceDnsCmd(t *testing.T, ns string, uncompress bool) *Command {
+	cmdArgs := ""
```

Comment:
perhaps `newTraceDnsCmd` can accept cmdArgs directly? So we can easily extend this in the future, for instance, to test ipv6.